kolibri/auth/migrations/0001_initial.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
```

Comment:
I ended up needing to delete the old migrations and create a new one, as it was getting some weird issues with migrating the changed uniqueness constraints. You can skip over this file and the next few deleted files; sorry for the bloat.